dips/dip-7.md

```diff
@@ -14,40 +14,56 @@ This DIP outlines the process by which technical working groups (TWG’s) are fo
 
-If there are material changes to the TWG’s scope, timeline, or expected output, a revised proposal must be submitted to the TSC for re-approval.
+If there are highly material changes to the TWG’s scope, a revised proposal must be submitted to the TSC for re-approval.
 
-Any TWG member can propose any topic or idea within the scope of the TWG up for discussion with the group. These discussions should be documented, and as the group comes to agreement on how to proceed, there should be clear ownership among TWG members for drafting, editing, obtaining feedback, and finalizing the group’s working product. There does not need to be total agreement in the TWG on the final output, but a supermajority (67%+) vote in favor is required before proceeding.
+Any TWG member can propose any topic or idea within the scope of the TWG up for discussion with the group. These discussions should be documented, and as the group comes to agreement on how to proceed, there should be clear ownership among TWG members for drafting, editing, obtaining feedback, and finalizing the group’s working product. A TWG can have any number of outputs.There must be rough consensus in the TWG on each final work product before proceeding. 
 
-Once the TWGs final output is ready and the group votes to proceed, it must be shared with the TSC for review and edits. After the editing process, the TSC must approve the output before it can be circulated within the Association.
+Once rough consensus is found, a work product must be shared with the TSC for review and edits. After the editing process, the TSC must approve the output via a majority vote. The work product can then be officially shared with the Association as final.
 
-If the output is a DIP, the TWG will submit the draft DIP as a pull-request to the diem/dip folder, entering the official [DIP process](https://dip.diem.com/overview/). At this point the DIP gets a DIP Manager to navigate the process from becoming a Draft DIP to a committed DIP in Github. 
+If the output is a DIP, the TWG will submit the draft DIP as a pull-request to the Diem/DIPs folder, entering the official DIP process. At this point the DIP gets a DIP Manager to navigate the process from becoming a Draft DIP to a committed DIP in Github. 
 
 In addition to assisting with the DIP process, a DIP Manager can also dedicate engineering resources to develop an implementation of the DIP if one does not exist already. This may mean: 
-Diem Core changes (via github issues and PRs)
-Move Framework changes (same implementation as above)
-Non-blockchain changes in which case things like off-chain work may be affected
+- Diem Core changes (via github issues and PRs)
+- Diem Framework changes (same implementation as above)
+- Non-blockchain changes in which case things like off-chain work may be affected
```

Comment:
Perhaps add a sentence "anything which is deemed relevant to the Diem project and the standard specification", since it already says (pretty much) everything which is blockchain or not blockchain, hence ... everything.